cumulusci/tasks/marketing_cloud/__init__.py

```diff
@@ -0,0 +1,3 @@
+from cumulusci.tasks.marketing_cloud.base import BaseMarketingCloudTask
+
+__all__ = (BaseMarketingCloudTask,)
```

Comment:
Leave this file out; tasks should be imported directly from the module they're defined in.